cmd/kops/create_cluster.go

```diff
@@ -475,7 +475,7 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 
 		// Default to a DNS name for the bastion
 		cluster.Spec.Topology.Bastion = &api.BastionSpec{
-			BastionPublicName: "bastion-" + clusterName,
+			BastionPublicName: "bastion." + c.DNSZone,
```

Comment:
One of the yaml files in https://github.com/kubernetes/kops/tree/master/tests/integration/create_cluster/private needs an update, and thus is failing.